setup/gcp/scripts/initialize_forseti_services.sh

```diff
@@ -30,25 +30,22 @@ set -o nounset
 # script will fail immediately rather than partially succeeding.
 echo "Cloud SQL Instance Connection string: ${SQL_INSTANCE_CONN_STRING}"
 echo "SQL port: ${SQL_PORT}"
-echo "Forseti DB name: ${FORSETI_DB_NAME}"
-echo "G Suite admin email: ${GSUITE_ADMIN_EMAIL}"
-echo "Root resource ID: ${ROOT_RESOURCE_ID}"
 if ! [[ -f $GSUITE_ADMIN_CREDENTIAL_PATH ]]; then
```

Comment:
I think we no longer have variable $GSUITE_ADMIN_CREDENTIAL_PATH set during the VM initialization.

Message I got during the init: 
./setup/gcp/scripts/initialize_forseti_services.sh: line 33: GSUITE_ADMIN_CREDENTIAL_PATH: unbound variable

Reply:
Thanks, fixed.